src/examples/confgure_npm_for_heroku.yml

```diff
@@ -0,0 +1,27 @@
+description: >
+  Configure npm for a Heroku nodejs applicaton and install deploy to Heroku.
+usage:
+  version: 2.1
+  orbs:
+    cloudsmith-npm: ft-circleci-orbs/cloudsmith-npm@1.1
+  jobs:
+    build:
+      docker:
+        - image: cimg/node:20.12
+      steps:
+        - checkout
+        - heroku/install
+        - cloudsmith-npm/configure_npm_for_heroku:
+            repository: "circleci-orb-testing"
+            app_name: "heroku-cloudsmith"
+            working_directory: ".circleci/heroku-cloudsmith-circleci-test"
```

Comment:
Should the working dir be removed from the example to avoid people copying and pasting and getting confused.

Reply:
Thanks, didn't realise i left this in. Will remove 👍